### PR TITLE
added svg call into menu walker and style init on front-end

### PR DESCRIPTION
**Pull request by PavelK27**
#26

### CODE REVIEW

File: assets/css/user.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.menu-item svg.icon {
+    height: 14px;
+    margin-right: 5px;
+    position: relative;
+    top: 2px;
+    width: 14px;
+}
```
Comment (phatsk) [inline assets/css/user.css]: Nitpick, but add a newline here.

**Review by phatsk — CHANGES_REQUESTED** [review]
Just the nitpick, otherwise this looks good.

**Review by phatsk — APPROVED** [review]
Looks good!

File: assets/css/user.css (hunk 1 above)
Comment (PavelK27) [reply assets/css/user.css]: added https://github.com/WebDevStudios/WDS-Mega-Menu/commit/fd396b42fdc3b874b30f1de2e484fe255cbf67ba

**Pull request merged**